.gitignore

```diff
@@ -42,3 +42,5 @@ _trial_temp*/
 /docs/build/
 /htmlcov
 /pip-wheel-metadata/
+
+.venv*
```

Comment:
It'd be nice to also match the `/xxx/` form above, unless you meant to match on multiple possible directory names.

Reply:
i think `/.venv*/` would still work, then, lemme try